### ISSUE TITLE
Game crash "Attempt to access invalid address"

### ISSUE THREAD

**Issue by FireController1847** [bloodypenguin/Skylines-ModTools#19]
And I know this is from Mod Tools only because it doesn't crash after I unsubscribe. When I re-subscribe, it works until I restart the game (which is required) in which it crashes.

```
========== OUTPUTING STACK TRACE ==================

  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A95A6668)
0x00007FF6A95A6668 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A95A6DF6)
0x00007FF6A95A6DF6 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A95E4DB3)
0x00007FF6A95E4DB3 (Cities) 
0x00000000371452D7 (Mono JIT Code) (wrapper managed-to-native) UnityEngine.Font:RequestCharactersInTexture (string,int,UnityEngine.FontStyle)
0x000000003714507B (Mono JIT Code) ColossalFramework.UI.UIDynamicFont:RequestCharacters (string,int,UnityEngine.FontStyle)
0x0000000037144D8E (Mono JIT Code) ColossalFramework.UI.UIDynamicFont/DynamicFontRenderer:MeasureString (string)
0x0000000037148D7D (Mono JIT Code) ColossalFramework.UI.UILabel:Invalidate ()
0x000000003714E244 (Mono JIT Code) ColossalFramework.UI.UIComponent:OnIsEnabledChanged ()
0x000000003714E380 (Mono JIT Code) ColossalFramework.UI.UIComponent:OnIsEnabledChanged ()
0x0000000037134FB4 (Mono JIT Code) ColossalFramework.UI.UIComponent:OnEnable ()
0x0000000037134D1E (Mono JIT Code) ColossalFramework.UI.UIPanel:OnEnable ()
0x0000000005FE08CB (Mono JIT Code) (wrapper runtime-invoke) object:runtime_invoke_void__this__ (object,intptr,intptr,intptr)
0x00007FFA01E25CDF (mono) mono_set_defaults
0x00007FFA01D78499 (mono) mono_runtime_invoke
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A9324954)
0x00007FF6A9324954 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A931FC2A)
0x00007FF6A931FC2A (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A95284C7)
0x00007FF6A95284C7 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A952B8B9)
0x00007FF6A952B8B9 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A9528F52)
0x00007FF6A9528F52 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A932EB5E)
0x00007FF6A932EB5E (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A8E8EBAB)
0x00007FF6A8E8EBAB (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A8E8EE06)
0x00007FF6A8E8EE06 (Cities) 
0x000000003713429B (Mono JIT Code) (wrapper managed-to-native) UnityEngine.GameObject:SetActive (bool)
0x00000000371893B3 (Mono JIT Code) ColossalFramework.UI.UIComponent:AttachUIComponent (UnityEngine.GameObject)
0x00000000A0C58758 (Mono JIT Code) DebugOutputPanel:AddMessageInternal (ColossalFramework.Plugins.PluginManager/MessageType,string)
0x000000000605D3A4 (Mono JIT Code) DebugOutputPanel:AddMessage (ColossalFramework.Plugins.PluginManager/MessageType,string,bool)
0x000000000605D028 (Mono JIT Code) DebugOutputPanel:AddMessage (ColossalFramework.Plugins.PluginManager/MessageType,string)
0x00000000A0C58695 (Mono JIT Code) ModTools.Log:Message (string)
0x00000000A0C58432 (Mono JIT Code) ModTools.ModTools:OnApplicationLogMessageReceivedThreaded (string,string,UnityEngine.LogType)
0x00000000A0C58560 (Mono JIT Code) (wrapper delegate-invoke) UnityEngine.Application/LogCallback:invoke_void__this___string_string_LogType (string,string,UnityEngine.LogType)
0x0000000006066719 (Mono JIT Code) UnityEngine.Application:CallLogCallback (string,string,UnityEngine.LogType,bool)
0x00000000060667CD (Mono JIT Code) (wrapper runtime-invoke) <Module>:runtime_invoke_void_object_object_int_sbyte (object,intptr,intptr,intptr)
0x00007FFA01E25CDF (mono) mono_set_defaults
0x00007FFA01D78499 (mono) mono_runtime_invoke
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A9324954)
0x00007FF6A9324954 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A931FC2A)
0x00007FF6A931FC2A (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A9320CEE)
0x00007FF6A9320CEE (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A94D01C2)
0x00007FF6A94D01C2 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A91923D9)
0x00007FF6A91923D9 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A9192A44)
0x00007FF6A9192A44 (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A9192C4C)
0x00007FF6A9192C4C (Cities) 
  ERROR: SymGetSymFromAddr64, GetLastError: 'Attempt to access invalid address.' (Address: 00007FF6A94D7D14)
0x00007FF6A94D7D14 (Cities) 
0x0000000006001F48 (Mono JIT Code) (wrapper managed-to-native) UnityEngine.DebugLogHandler:Internal_Log (UnityEngine.LogType,string,UnityEngine.Object)
0x0000000006001E4C (Mono JIT Code) UnityEngine.DebugLogHandler:LogFormat (UnityEngine.LogType,UnityEngine.Object,string,object[])
0x0000000006001BDC (Mono JIT Code) UnityEngine.Logger:Log (UnityEngine.LogType,object)
0x0000000006001990 (Mono JIT Code) UnityEngine.Debug:Log (object)
0x00000000060AB44E (Mono JIT Code) PopsManager:LogCallback (string)
0x00000000A0C58282 (Mono JIT Code) PopsApi.PopsApiWrapper:LogCallbackImpl (intptr,intptr,string)
0x00000000060AB0E9 (Mono JIT Code) (wrapper native-to-managed) PopsApi.PopsApiWrapper:LogCallbackImpl (intptr,intptr,intptr)
0x00007FF9E5B3700A (pops_api) POPS_TrustLevelRetrieveCurrent
0x00007FF9E5B91B37 (pops_api) pdx_http_url_encode
0x00007FF9E5B9424B (pops_api) pdx_http_url_encode
0x00007FF9E5B3A8A9 (pops_api) POPS_TrustLevelRetrieveCurrent
0x00000000060CF89C (Mono JIT Code) (wrapper managed-to-native) PopsApi.PopsApiWrapper:POPS_RunCallbacks (intptr)
0x00000000060CF7F7 (Mono JIT Code) PopsApi.PopsApiWrapper:Update ()
0x00000000060CF743 (Mono JIT Code) PopsApi.PopsApiWrapper:OnTimeoutUpdate (object,System.Timers.ElapsedEventArgs)
0x00000000060CF03C (Mono JIT Code) System.Timers.Timer:Callback (object)
0x0000000005FD0126 (Mono JIT Code) (wrapper runtime-invoke) object:runtime_invoke_void__this___object (object,intptr,intptr,intptr)
0x00007FFA01E25CDF (mono) mono_set_defaults
0x00007FFA01D78499 (mono) mono_runtime_invoke
0x00007FFA01D7E9A7 (mono) mono_runtime_invoke_array
0x00007FFA01D7EC7C (mono) mono_runtime_invoke_array
0x00007FFA01D9F0EB (mono) mono_security_set_mode
0x00007FFA01D9FB42 (mono) mono_security_set_mode
0x00007FFA01DA388F (mono) mono_thread_interruption_request_flag
0x00007FFA01E58955 (mono) unity_mono_reflection_method_get_method
0x00007FFA3FCA3034 (KERNEL32) BaseThreadInitThunk
0x00007FFA42111431 (ntdll) RtlUserThreadStart

========== END OF STACKTRACE ===========
```

**Comment by FireController1847** [comment]
@bloodypenguin Any idea what's going on? I really need to use this mod but I can't...

**Comment by FireController1847** [comment]
This seems to only happen after you log in. It launches fine before I log in, and the second I hit the login button it crashes and continues crashing on launch.

**Comment by FireController1847** [comment]
Changing this back to true fixed the issue...
![image](https://user-images.githubusercontent.com/16494272/44164452-54e12000-a083-11e8-9a54-5417f644f615.png)

**Comment by dymanoid** [comment]
This is fixed with 26214166b29551c741431b4c797185eecb53b0bb.
